code/modules/projectiles/guns/flamer.dm

```diff
@@ -69,6 +69,8 @@
 	var/lit_overlay_offset_y = 0
 	///Damage multiplier for mobs caught in the initial stream of fire.
 	var/mob_flame_damage_mod = 2
+	///how wide of a cone the flamethrower produces on wide mode.
+	var/cone_angle = 55
```

Comment:
Should this be on the nozzle itself rather than the gun?

Reply:
There's no other nozzles that change it, and if there was then it would make sense for it to be a modifier to the base gun's angle.

Reply:
I kinda agree with QV here

Reply:
The reason why I want to have it on the gun is so that I can add one or more new flamers in the future that different angles, while still being compatible with the sam nozzles.